lambdatimes/src/components/Content/Tab.js

```diff
@@ -1,32 +1,32 @@
 import React from "react";
 import PropTypes from "prop-types";
 
-const Tab = ({ tab, selectTabHandler}) => {
-  // Make sure you include PropTypes on your props.
-  Tab.propTypes = {
-    tab: PropTypes.string,
-    selectTabHandler: PropTypes.func
-  };
+const Tab = props => {
 
   /* Using your props, determine if the `tab` prop matches the `selectedTab` prop, 
       if they match, the className should be: 'tab active-tab', 
       if it is not it should just be 'tab'*/
 
-  
   return (
-    <div className="tab active-tab"
-      //if tab === selectedtab { return className=" active-tab"}
-     // else return  className="tab"
+    <div 
+    className={props.selectedTab === props.tab ? 'tab active-tab' : 'tab'}
```

Comment:
Great choice of code for this line 👍 Could you explain why you chose this particular way of doing the code Antonio?